src/webui/web_server_utils/system_monitor_mixin.py

```
@@ -57,6 +59,34 @@ def restart_backend(self) -> tuple[dict, int]:
+
+        Returns:
+            tuple[dict, int]: Acceptance or error payload with HTTP status.
+        """
+        if sys.platform != "linux":
+            return {"error": "System reboot is only supported on Linux."}, 400
+
+        def execute_reboot() -> None:
+            time.sleep(0.5)
+            try:
+                subprocess.Popen(
+                    ["sudo", "reboot"],
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)